CIScripts/Test/TestConfigurationUtils.ps1

```diff
@@ -109,34 +117,25 @@ function Start-DockerDriver {
         "-os_tenant_name", $OpenStackConfig.Project,
         "-adapter", $AdapterName,
         "-vswitchName", "Layered <adapter>",
+        "-logPath", $OldLogPath,
         "-logLevel", "Debug"
     )
 
     Invoke-Command -Session $Session -ScriptBlock {
 
-        $LogDir = "$Env:ProgramData/ContrailDockerDriver"
-
-        if (Test-Path $LogDir) {
-            Push-Location $LogDir
-
-            if (Test-Path log.txt) {
-                Move-Item -Force log.txt log.old.txt
-            }
-
-            Pop-Location
-        }
-
         # Nested ScriptBlock variable passing workaround
         $Arguments = $Using:Arguments
+        $LogDir = $Using:LogDir
 
         Start-Job -ScriptBlock {
-            Param($Arguments)
-            & "C:\Program Files\Juniper Networks\contrail-windows-docker.exe" $Arguments
+            Param($Arguments, $LogDir)
+
+            New-Item -ItemType Directory -Force -Path $LogDir | Out-Null
+            $LogPath = Join-Path $LogDir "contrail-windows-docker-driver.log"
+            $ErrorActionPreference = "Continue"
 
-            # The `, $null` below is used to force passing $Arguments as an arguments'
-            # list element instead of an arguments list itself.
-            # @($Arguments) looks like it should work, but does not.
-        } -ArgumentList $Arguments, $null
+            & "C:\Program Files\Juniper Networks\contrail-windows-docker.exe" $Arguments 2>> $LogPath
```

Comment:
Let's try this here:
`& "C:\Program Files\Juniper Networks\contrail-windows-docker.exe" $Arguments 2>&1 | Out-File -Append -Width 4096 $LogPath`